salt/modules/mssql.py

```diff
@@ -67,7 +71,71 @@ def _get_connection(**kwargs):
             connection_args[arg] = __salt__["config.option"](
                 "mssql." + arg, _DEFAULTS.get(arg, None)
             )
-    return pymssql.connect(**connection_args)
+    conn = pymssql.connect(**connection_args)
+    conn.autocommit(True)
```

Comment:
Since you're setting `autocommit(True)` here it worth to update all the functions using `_get_connection`.

Reply:
Also `autocommit` could be passed as an argument of `pymssql.connect` so I propose to better put it into args:
```
connection_args['autocommit'] = True
conn = pymssql.connect(**connection_args)
```

Reply:
Do you mean it is up to the user to decide whether to open autocommit?
Then set mssql.autocommit: 'true' in the minion configuration file?

Reply:
I don't think so. I'd like to set it after we read all the user input right before `pymssql.connect` is called.

Reply:
Ok, I see